tests/suite/snippets_filter_test.go

```diff
@@ -79,29 +70,44 @@ var _ = Describe("SnippetsFilter", Ordered, Label("functional", "snippets-filter
 			for _, name := range snippetsFilterNames {
 				nsname := types.NamespacedName{Name: name, Namespace: namespace}
 
-				err := waitForSnippetsFilterToBeAccepted(nsname)
+				err := waitForSnippetsFilterToBeAccepted(
+					nsname,
+					metav1.ConditionTrue,
+					ngfAPI.SnippetsFilterConditionReasonAccepted,
```

Comment:
This seems redundant to add these args, the function can just check for them since we aren't using the function for any other conditions, right?